handlers/validator.go

```diff
@@ -183,21 +183,13 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 				}
 			}
 
-			sumValid := uint64(0)
 			// check if a valid deposit exists
```

Comment:
**Minor:** This comment is wrong, it should be
```suggestion
			// check if an invalid deposit exists
```

Reply:
Changed.